### PR TITLE
[python] JSON format for rolling batch

### PR DESCRIPTION
**Pull request by xyang16**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Review by lanking520 — APPROVED** [review]

File: engines/python/src/main/java/ai/djl/python/engine/RollingBatch.java [hunk 1]
```diff
@@ -179,7 +179,10 @@ void addResponse(String json, boolean enableStreaming) {
             } else {
                 nextToken.append(element.get("data").getAsString());
                 if (last) {
-                    data.appendContent(BytesSupplier.wrap(nextToken.toString()), true);
+                    data.appendContent(
```
Comment (frankfliu) [inline engines/python/src/main/java/ai/djl/python/engine/RollingBatch.java:182]: This doesn't really work if we only make last token to json.

File: .github/workflows/continuous.yml [hunk 2]
```diff
@@ -38,10 +38,12 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
       - name: Build with Gradle
-        run: ./gradlew --refresh-dependencies build :jacoco:testCodeCoverageReport
```
Comment (frankfliu) [inline .github/workflows/continuous.yml:41]: Why only build for python engine?
This workflow is used to verify the whole repo

**Review by frankfliu — CHANGES_REQUESTED** [review]

File: .github/workflows/continuous.yml (hunk 2 above)
Comment (xyang16) [reply .github/workflows/continuous.yml]: I was debugging the jacoco report error. It's not going to be commited.

File: engines/python/setup/djl_python/rolling_batch/rolling_batch.py [hunk 3]
```diff
@@ -24,16 +36,23 @@ class Request(object):
 
     """
 
-    def __init__(self, id: int, input_text: str, parameters: dict):
+    def __init__(self,
+                 id: int,
+                 input_text: str,
+                 parameters: dict,
+                 output_formatter=_default_output_formatter):
```
[the remark this answers is not in the document]
Comment (xyang16) [reply engines/python/setup/djl_python/rolling_batch/rolling_batch.py:43]: https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/setup/djl_python/encode_decode.py#L75

[the remark this answers is not in the document]
Comment (xyang16) [reply engines/python/setup/djl_python/rolling_batch/rolling_batch.py:43]: Changed.

**Review by frankfliu — APPROVED** [review]

**Pull request merged**